readme.md

```diff
@@ -32,14 +32,14 @@ cpy(['src/*.png'], 'dist', function (err) {
 
 #### files
 
-*Required*  
+*Required*
 Type: `array`
 
 Files to copy.
 
 #### destination
 
-*Required*  
```

Comment:
Installing an EditorConfig plugin in your editor would have prevented this ;)

https://github.com/sindresorhus/cpy/blob/58437de6de568b6e4759a4e596aa74cb45d70344/.editorconfig#L15